src/forklift/models.py

```diff
@@ -233,6 +233,16 @@ def add_packing_slip(self, slip):
         for slip_crate in slip['crates']:
             crate_objects_by_name[slip_crate['name']].result = [slip_crate['result']]
 
+    def init_standalone(self):
```

Comment:
```suggestion
    def configure_standalone_logging(self):
```

Reply:
My thought was that eventually, it may have more than just logging in it. But I'm good with whatever.